apps/nextra/pages/en/network/blockchain/aptos-white-paper.mdx

```diff
@@ -33,6 +33,14 @@ import { Card, Cards} from "@components/index";
       Aptos White Paper PDF translated to Japanese
     </Card.Description>
   </Card>
+  <Card href="https://aptosfoundation.org/whitepaper/aptos-whitepaper_es.pdf">
+    <Card.Title linkType={null}>
+      🇪🇸 Español Spanish (es-ES) ↗
```

Comment:
We may want to check it later, I believe it's probably translated to Latin American Spanish (or US Spanish) rather than Spain Spanish

Reply:
I confirmed that there is really not a big difference for LATAM Spanish and Spain Spanish. But we can display Spain flag on this page